torch/multiprocessing/spawn.py

```diff
@@ -1,14 +1,22 @@
 
 from typing import Optional
+import logging
+import mmap
 import multiprocessing
 import multiprocessing.connection
+import os
 import signal
 import sys
 import warnings
 
 from . import _prctl_pr_set_pdeathsig  # type: ignore[attr-defined]
 
 
+logger = logging.getLogger(__name__)
+
+RANK_STATUSES_FILE_NAME = "rank_statuses"
```

Comment:
I think we can define this as a constant in cpp and expose it via pybind: https://pybind11-jagerman.readthedocs.io/en/stable/advanced.html#exporting-constants-and-mutable-objects